### ISSUE TITLE
Soul Hackers 2 (1777620)

### ISSUE THREAD

**Issue by crebwyllion** [ValveSoftware/Proton#6139]
# Compatibility Report
- Name of the game with compatibility issues: Soul Hackers 2
- Steam AppID of the game: 1777620

## System Information
- GPU: AMD Radeon RX 5700
- Driver/LLVM version: Mesa 22.1.7
- Kernel version: 5.19.6-200.fc36.x86_64
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/crebwyllion/5625297796fd914f46e81265da6d51f5
- Proton version: Proton Experimental

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
Certain spell effects, textures, and other such things are missing, Ringo's blade slash outside of combat being the easiest to test and see. Other people have also reported issues with fullscreen, but that's outside the purview of this report.

[steam-1777620.log](https://github.com/ValveSoftware/Proton/files/9484280/steam-1777620.log)
## Reproduction
Load the game, start a new file, and play through until you reach a scene where it's suppose to be raining ([shown here](https://youtu.be/I7IFlEJ5eM4?t=1015)). There will be water dripping down the screen, but no raindrops ([shown here](https://user-images.githubusercontent.com/85197579/188305343-bd755602-4400-43f1-aae4-33d04a2637d7.png)). A more obvious example would be a cutscene not much later where there should be triangles floating across the screen ([shown here](https://youtu.be/I7IFlEJ5eM4?t=1578)), but there won't be on Proton ([shown here](https://user-images.githubusercontent.com/85197579/188305309-e0adddac-b11f-4716-90f6-11b965e48a66.png)).

**Comment by Blisto91** [comment]
@crebwyllion If the game is able to launch with it could try and see if the effects are there with `PROTON_USE_WINED3D=1 %command%`

**Comment by crebwyllion** [comment]
They do not appear with WineD3D.

**Comment by Blisto91** [comment]
Thanks for checking.
Do you know if this issue is also present with proton 7.0-4 or 6.3-8?
If those versions work.

Edit: oh wait i misread. They do also not work with wined3d

**Comment by crebwyllion** [comment]
Oh, I meant the missing effects do not appear with WineD3D, yeah.

I guess you don't need me to anymore, but I tested with both DXVK and WineD3D on both Proton versions you mentioned and they also didn't work. Just in case it helps?

**Comment by waspennator** [comment]
I feel as though the deck playable status should be updated to note there are visual issues, for some reason it's not mentioned at all.

**Comment by LynnAlice** [comment]
Still unresolved. Please fix it, because the game is now listed as playable, but the missing effects greatly impact how the game feels

**Comment by K0bin** [comment]
Turns out this is a Wine bug in D3DCompiler.

**Comment by LynnAlice** [comment]
Issue can be resolved with installing d3dcompiler-47 with protontricks

**Comment by kisak-valve** [comment]
Soul Hackers 2 (1777620)

Issue transferred from https://github.com/ValveSoftware/Proton/issues/9215.
@LowSkillDeveloper posted on 2025-11-13T10:25:59:


The game doesn't work on new versions of Proton, including Proton 10.0-2 and Experimental.
However, the game runs fine on older versions, works on version 9.0-4

PROTON_LOG of Proton 10.0-2:
[steam-1777620.log](https://github.com/user-attachments/files/23522300/steam-1777620.log)

# Compatibility Report
- Name of the game with compatibility issues: Soul Hackers 2
- Steam AppID of the game: 1777620
- The game is marked as compatible with Steam Deck, with a note about the presence of small text in the game.


## System Information
Steam Deck OLED 1TB
- Video driver version: Mesa 25.2.0
- Kernel version: 6.16.12-valve2-1-neptune-616-g8a732b312b58
- Proton version: 10.0-2 & Experimental

## I confirm:
- that I haven't found an existing compatibility report for this game, just another report not related to the current issue: https://github.com/ValveSoftware/Proton/issues/6139
- that I have checked whether there are updates for my system available.

## Symptoms

When launching the game, it freezes, a loading icon briefly appears in the game, after which the company logos should show up, but they don’t. The game stops working, and even the FPS counter freezes. The issue persists on Proton 10, but there’s no problem on Proton 9. 


## Reproduction

1. Download the game https://store.steampowered.com/app/1777620/Soul_Hackers_2/
2. Install Proton 10 or experimental version.
3. Launch the game
4. The game freezes while startup loading. And it doesn't respond anymore, even the fps counter freezes.

**Comment by Blisto91** [comment]
@LowSkillDeveloper Thank you for the report. This should now be working again on [Proton Bleeding Edge](https://github.com/ValveSoftware/Proton/wiki/Proton-Versions#proton-bleeding-edge)

**Comment by LowSkillDeveloper** [comment]
> This should now be working again on [Proton Bleeding Edge](https://github.com/ValveSoftware/Proton/wiki/Proton-Versions#proton-bleeding-edge)

Yes, the latest version of Proton Bleeding Edge launches the game just like before.
